### PR TITLE
Start to use GitHub Action

### PR DESCRIPTION
**Pull request by mssun**
Replace Travis CI to GitHub Action.

### CODE REVIEW

**Comment by mssun** [comment]
@SimplyDanny, do you know why this case will randomly failed in CI?

```
passKitTests.GitCredentialTest
  testSSHKeyCredentialProvider, XCTAssertNotNil failed
  /Users/runner/work/passforios/passforios/passKitTests/Models/GitCredentialTest.swift:81
  

        XCTAssertNotNil(credentialProvider.credential(for: .sshCustom, url: nil, userName: nil))
        XCTAssertNil(credentialProvider.credential(for: .sshCustom, url: nil, userName: nil))
```
(https://github.com/mssun/passforios/pull/432/checks?check_run_id=1388966988#step:5:991)

**Comment by SimplyDanny** [comment]
Did you observe this flakiness before or is it new with the GitHub Action?

Why it fails is not clear for me. It runs fine locally. If you want to progress, just ignore the test case for the moment. The root cause can be investigated later.

File: .github/workflows/linting.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+on: pull_request
+
+jobs:
+  swiftformat:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Installing packages
+      run: brew update && brew install swiftformat
+    - name: swiftformt
```
Comment (SimplyDanny) [inline .github/workflows/linting.yml:12]:
```suggestion
    - name: swiftformat
```

File: scripts/swiftlint.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 SWIFTLINT_VERSION="0.41.*"
 
-if [[ -f "${SRCROOT}/.ci-env" ]]; then
+if [[  "${CI}" == "true" ]]; then
```
Comment (SimplyDanny) [inline scripts/swiftlint.sh:3]: Shouldn't it be the same for the SwiftFormat script?

**Review by SimplyDanny — COMMENTED** [review]
I wonder whether SwiftLint and SwiftFormat should actually run on CI. A problem can be that the versions in the scripts and the version of the installed tools are different. So CI could potentially find different issues. This is the reason why I introduced this `.ci-env` file to deactivate listing and formatting on CI.

**Pull request merged**